packages/haul-ram-bundle-webpack-plugin/src/__tests__/FileRamBundle.test.ts

```diff
@@ -0,0 +1,43 @@
+import { RawSource } from 'webpack-sources';
+import MAGIC_NUMBER from 'metro/src/shared/output/RamBundle/magic-number';
+import FileRamBundle from '../FileRamBundle';
+
+test('FileRamBundle should create valid RAM bundle', () => {
+  const bootstrapper = '(function() { /* bootstrap */ })()';
+  const modules = [
+    {
+      id: 0,
```

Comment:
likely not relevant to the test, but looks like `id` can be a string, maybe worth adding a case where it actually is a string

Reply:
it's tested in integration tests